extension/page-detect.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
```

Comment:
This *may* be different with content scripts, since they run in a separate execution context, but typically Chrome confines a global strict pragma to the individual resource (inline `<script>` tag or external JS file), if I recall. [Example](http://plnkr.co/edit/EvlE2erWykq2XHLcjkTK?p=preview)